zookeeper-server/src/main/java/org/apache/zookeeper/cli/RestoreCommand.java

```diff
@@ -145,7 +153,9 @@ public String getUsageStr() {
         + OptionFullCommand.LOCAL_RESTORE_TEMP_DIR_PATH
         + ": Optional, local path for creating a temporary intermediate directory for restoration, the directory will be deleted after restoration is done\n    "
         + OptionFullCommand.DRY_RUN + " " + OptionLongForm.DRY_RUN
-        + ": Optional, no files will be actually copied in a dry run";
+        + ": Optional, no files will be actually copied in a dry run\n    " + OptionFullCommand.OVERWRITE
+        + " " + OptionLongForm.OVERWRITE
+        + ": Optional, default false. If set to true, the destination directories will be overwritten if exist";
```

Comment:
"If true, the files in the destination directories will be overwritten." would suffice.

Also, another thing we should make clear: are we deleting existing files (if any) in the directory and populating it with restored files or are we just overwriting if there are any files with the same name?

Reply:
Discussed offline and reached an agreement. We will completed clear the directory and populate it with the restored files in this case. Also we will log the file names of the files that are deleted while clear out the directory. In the future, if necessary, we could consider moving the directory to a backup folder instead of clearing out the directory.